docs/references/subcommands/analyze.md

```diff
@@ -67,6 +67,14 @@ The Vendored Dependencies feature allows you to scan for licenses directly in yo
 | `--force-vendored-dependency-scan-method` | Force the vendored dependency scan method. The options are 'CLILicenseScan' or 'ArchiveUpload'. 'CLILicenseScan' is usually the default unless your organization has overridden this. |
 | `--force-vendored-dependency-rescans`     | Force vendored dependencies to be rescanned even if the revision has been previously analyzed by FOSSA. This currently only works for CLI-side license scans.                         |
 
+### Custom License Searches
```

Comment:
Could you link to this section directly somewhere in the TOC? https://github.com/fossas/fossa-cli/tree/master/docs#table-of-contents 

I know we already link to the analyze.md doc, but I think it's worth surfacing sections like this higher. I haven't gotten around to trying to rewrite the TOC but having more topics listed will make that work easier in the future. I wouldn't do anything fancy, just maybe dump the link somewhere?